src/test/java/com/jcabi/s3/AwsBucketITCase.java

```diff
@@ -147,4 +150,17 @@ public void listsInPrefixedBucketWithouCollisions() throws Exception {
         }
     }
 
+    /**
+     * AwsBucket can check if the bucket exists.
+     * @throws IOException If fails
+     */
+    @Test
+    public void checkBucketExistence() throws IOException {
+        final Bucket bucket = this.rule.bucket();
+        Assert.assertTrue(bucket.exists());
```

Comment:
@yegor256 what do you mean under "future tests will fail"? If I understand correctly, you mean that `exists()` either returns `false` or throws an exception. In any case test `checkBucketExistence` will fail and it seems there is no need in `try/finally`.